chapters/part3/bayesian_networks/index.html

```diff
@@ -44,26 +44,26 @@ <h2>Bayesian Networks</h2>
 &= \prod_i P(x_i | x_{i-1}, \dots, x_1)\\
 \end{align*}
 
-By looking at the difference in the two equations, we can see that a Bayes' Net is assumping that 
+By looking at the difference in the two equations, we can see that a Bayes' Net is assuming that 
 $$P(x_i | x_{i-1}, \dots, x_1) = P(x_i | \text{Values of parents of }X_i)$$
 This is a conditional independence statement. It is saying that once you know the value of the parents of a variable in your network, $X_i$, any further information about non-descendents will not change your belief in $X_i$. Formally we say that $X_i$ is conditionally independent of its non-descendents, given its parents. What is a non-descendent again? In a graph, a descendent of $X_i$ is anything which is in the subtree that starts at $X_i$. Everything else is a non-descendent. Non-descendents include the "ancestor" nodes of $X_i$ as well as nodes which are totally unconnected to $X_i$. When designing Bayes' Nets you don't have to think about this assumption directly. It turns out to be a naturally good assumption if the arrows between your nodes follow a causal path.
 </div>
 
-<h2>Designing a Bayes Net</h2>
+<h2>Designing a Bayes' Net</h2>
```

Comment:
Its not possessive. It is short for a Bayesian Network (perhaps the full name would be better)
https://en.wikipedia.org/wiki/Bayesian_network

Reply:
Got it, thanks for letting me know! Was definitely still thinking of Bayes' Theorem. Just fixed it in a new commit.